tests/ui/traits/vtable/vtable-dyn-incompatible.rs

```diff
@@ -1,13 +1,14 @@
-//@ build-fail
 #![feature(rustc_attrs)]
 
 // Ensure that dyn-incompatible methods in Iterator does not generate
 // vtable entries.
 
 #[rustc_dump_vtable]
-trait A: Iterator {}
+type Test = dyn A<Item=u8>;
 //~^ error vtable
 
+trait A: Iterator {}
```

Comment:
vibe, pls move below the trait definition
```suggestion
trait A: Iterator {}

#[rustc_dump_vtable]
type Test = dyn A<Item=u8>;
//~^ error vtable
```